pytensor/link/jax/dispatch/scalar.py

```diff
@@ -261,6 +285,14 @@ def psi(x):
     return psi
 
 
+@jax_funcify.register(TriGamma)
```

Comment:
is it dispatched on vector inputs?

Reply:
Yes, I added more comments in the `ScalarOp` dispatch to make it more understandable.